widgets/outputs/src/OutputsTable.js

```diff
@@ -38,12 +42,10 @@ export default class extends React.Component {
     render() {
         const NO_DATA_MESSAGE =
             "There are no Outputs/Capabilities available. Probably there's no deployment created, yet.";
-        const { DataTable, ErrorMessage, Header, Label } = Stage.Basic;
+        const { DataTable, ErrorMessage, Header, Button } = Stage.Basic;
```

Comment:
Let's keep it in alphabetical order.

```suggestion
        const { Button, DataTable, ErrorMessage, Header } = Stage.Basic;
```